kingpin/utils.py

```diff
@@ -71,13 +74,25 @@ def str_to_class(string):
     class_name = string_elements.pop()
     module_name = '.'.join(string_elements)
 
-    # load the module, will raise ImportError if module cannot be loaded
-    # or if that module has a failed import inside of it.
-    m = __import__(module_name, globals(), locals(), class_name)
-    # get the class, will raise AttributeError if class cannot be found
-    c = getattr(m, class_name)
+    for prefix in ['kingpin.actors', 'actors']:
+        try:
+            prefixed_module_name = "%s.%s" % (prefix, module_name)
+
+            # load the module, will raise ImportError if module cannot be
+            # loaded or if that module has a failed import inside of it.
+            m = __import__(prefixed_module_name, globals(), locals(),
+                           class_name)
 
-    return c
+            # get the class, will raise AttributeError if class cannot be found
+            return getattr(m, class_name)
+
+        except ImportError:
```

Comment:
At least print the error here. If `prefixed_module_name` and `class_name` are correct, but inside of that python file you import something that doesn't exist - it will still be `ImportError` and it will behave as if the original file failed to import. This can take quite a long time to debug